google-cloud-bigtable/lib/bigtable/instance_admin_client.rb

```diff
@@ -0,0 +1,376 @@
+# frozen_string_literal: true
+
+require "google/cloud/bigtable/admin/v2"
+require "google/bigtable/admin/v2/instance_pb"
+
+module Bigtable
+  # Protobuf references
+  Instance = Google::Bigtable::Admin::V2::Instance
+  Cluster = Google::Bigtable::Admin::V2::Cluster
+
+  class InstanceAdminError < StandardError; end # :nodoc:
```

Comment:
Where is this error class being used? This should probably inherit from `Google::Cloud::Error`.

Reply:
@blowmage Right now not in use. Removing it.